test/test_meme.py

```diff
@@ -0,0 +1,11 @@
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID, TEST_BOT_ID
+
+def test_meme(uqcsbot: MockUQCSBot):
+    '''
+    Test !meme
+    '''
+    uqcsbot.post_message(TEST_CHANNEL_ID, '!meme bender "top" "bottom"')
+    messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
+
+    assert len(messages) == 2
+    assert messages[1]['attachments'][0]['image_url'] == 'https://memegen.link/bender/top/bottom.jpg'
```

Comment:
prefer `messages[-1]` over `messages[1]` as it implies its the most recent message